orbslam3:
  ros__parameters:
    world_frame: "map_vslam"
    odom_frame: "odom_frame"
    camera_optical_frame: "d455_infra1_optical_frame"
    body_frame: "d455_link"
    body_optical_frame: "d455_imu_optical_frame"